### PR TITLE
Add display of latest update of company contact

### PR DESCRIPTION
**Pull request by jonasdeluna**
# Description

Add latest company contact update date display

# Result

If you've made visual changes, please check the boxes below and include images showing the changes. Descriptions are appreciated.

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

> [!CAUTION]
> Make sure your images do not contain any real user information.

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
</img>
        </td>
        <td>
        </img>
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Please describe what and how the changes have been tested, and provide instructions to reproduce if necessary.

---

Resolves: ABA-1055

### CODE REVIEW

**Review by falbru — APPROVED** [review]
Very nice :100:

File: app/routes/bdb/components/BdbDetail.tsx [hunk 1]
```diff
@@ -472,15 +476,21 @@ const BdbDetail = () => {
           />
 
           <Flex justifyContent="space-between" alignItems="center">
-            <h3>Bedriftskontakter</h3>
+            <Flex column>
```
Comment (ivarnakken) [inline app/routes/bdb/components/BdbDetail.tsx:479]: I don't think this is needed

File: app/routes/bdb/components/BdbDetail.tsx [hunk 2]
```diff
@@ -472,15 +476,21 @@ const BdbDetail = () => {
...
               Legg til bedriftskontakt
             </LinkButton>
           </Flex>
           {company.companyContacts?.length > 0 ? (
             <Table
               columns={contactsColumns}
-              data={company.companyContacts}
+              data={company.companyContacts.map((contact) => ({
+                ...contact,
+                updatedAt: moment(contact.updatedAt).format('YYYY-MM-DD'),
```
Comment (ivarnakken) [inline app/routes/bdb/components/BdbDetail.tsx:492]: Isn't it better to add this to the render column prop? Like we usually do

File: app/routes/bdb/components/BdbDetail.tsx [hunk 3]
```diff
@@ -439,6 +444,28 @@ const BdbDetail = () => {
...
+            <Table
+              columns={contactsColumns}
+              data={company.companyContacts.map((contact) => ({
+                ...contact,
+                updatedAt: moment(contact.updatedAt).format('YYYY-MM-DD'),
+              }))}
+              hasMore={false}
+              loading={showSkeleton}
+            />
+          )}
```
[the remark this answers is not in the document]
Comment (jonasdeluna) [reply app/routes/bdb/components/BdbDetail.tsx]: I think it's been fixed now!

File: app/routes/bdb/components/BdbDetail.tsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ import {
 import { usePreparedEffect } from '@webkom/react-prepare';
 import { isEmpty } from 'lodash';
 import { Trash2 } from 'lucide-react';
+import moment from 'moment';
```
Comment (ivarnakken) [inline app/routes/bdb/components/BdbDetail.tsx:14]: hehe we actually use `moment-timezone`

```suggestion
import moment from 'moment-timezone';
```

Comment (jonasdeluna) [reply app/routes/bdb/components/BdbDetail.tsx]: Fixed this now :+1:

**Review by ivarnakken — APPROVED** [review]
Very clean!😍🤩

**Pull request merged**